### ISSUE TITLE
Make Attach Visual Studio Code a higher level button

### ISSUE THREAD

**Issue by servient-ashwin** [microsoft/vscode-remote-release#10228]
Current workflow:

1. Go to the docker container extension.
2. Expand `Containers` tab.
3. Among the list of containers, choose one, right click on the container.
4. Click on `Attach Visual Studio Code` menu item from the menu.
5. Choose your container from the top menu pop up that opens up.
6. Visual studio code is now attached.

Expected workflow scenario 1:

1. Go to the docker container extension.
2. Expand `Containers` tab.
3. Among the list of containers, choose one, right click on the container.
4. Click on `Attach Visual Studio Code` menu item from the menu.
5. Visual studio code is now attached.

Expected workflow scenario 2:

1. Go to the docker container extension.
2. Expand `Containers` tab.
3. Click on the global `Attach to Visual Studio Code` button.
4. Choose your container from the top menu pop up that opens up.
5. Visual studio code is now attached.

At the moment, you need to select the container twice to attach it to VSCode, once when you right click (on the container you want to attach) and click attach and then again when the top menu pop-up opens and you choose the same container again. This is counter intuitive 
1. If I right clicked on the container I want open why is there an option to choose it again?
2. If the option is indeed necessary because reasons, why do individual containers have a attach option on right click, make it global anyway.

With the amount of clicks saved by making this one change, you can save a developer 1s everytime they would like to do this.

If you consider the latest stackoverflow developer survey to be any honest indication of what people actually do
![image](https://github.com/user-attachments/assets/b1e0b8f8-e22c-481b-943b-f5606ce0e4d4)

You will see ~76% of the respondents use VScode with additional ~4% using codium. If we round up figures for math purposes (and because I am not that good at math) ~80% of 60k respondents use VSCode and such which boils down to around 50000 users.

Now giving in to the paranoia and considering that 25% (this is scarily huge, but it math the maths) use this extension daily and use the current workflow and spend a second and assuming they use it only once (wild assumption but because so far you know me as a math prodigy) they spend 12,500s just clicking stuff they already knew about.

Which means
12500s -> ~3.5hrs daily -> 1277hrs yearly -> 53 days just to click a tab.
Imagine the time saved (probably doing nothing productive anyways, but that's not the point, or maybe, but let's skip that).
This reminds me of 

![image](https://github.com/user-attachments/assets/a88d60e7-de7b-4997-b46d-37530748c387)

TLDR: Can we make this simple?

PS: If this turns out to just be a me problem and you already have a button/ easier way that does this or if this is the wrong repo, please excuse my ignorance and allow me to slide under the desk and never think about this again.

**Comment by chrmarti** [comment]
Not sure I follow the math, but you can use `F1` > `Dev Containers: Attach to Running Container...` as a shortcut.

You can also go through the Remote Explorer (instead of the Docker view) that does not have the problem of having to select the container twice.

The bug for having to select the container twice when going through the Docker view while connected to a remote host is tracked as https://github.com/microsoft/vscode-remote-release/issues/4405.
